scripts/e2e.sh

```diff
@@ -0,0 +1,71 @@
+MY_PATH="`dirname \"$0\"`"              # relative
+MY_PATH="`( cd \"$MY_PATH\" && pwd )`"  # absolutized and normalized
+if [ -z "$MY_PATH" ] ; then
+  # error; for some reason, the path is not accessible
+  # to the script (e.g. permissions re-evaled after suid)
+  exit 1  # fail
+fi
+
+
+# check for docker
+DOCKER_RESP=$(docker version)
+if grep -q "Is the docker daemon running?" <<< "$DOCKER_RESP"; then
+  echo "You need to start the docker daemon"
+  exit 1 #fail
+fi
+
+# check for jq
+JQ_RESP=$(which jq)
+if [ ${#JQ_RESP} -eq 0 ]; then
+  echo "You need to install the jq package (brew install jq)"
+  exit 1 #fail
+fi
+
+
+SPIN_UP=false
+
+while true; do
+    read -p "Do you wish to build containers? (default=no) " yn
+    case $yn in
+        [Yy]* ) SPIN_UP=true;break;;
+        [Nn]* ) break;;
+        * ) break;;
+    esac
+done
+
+echo $SPIN_UP;
+
+
+export PORTAL_SOURCE_DIR=$PWD;
```

Comment:
for all of these exports, what I plan to do is only set them to these defaults if the are NOT already set.  that way people can configure them in custom fashion as necessary.  most people will just rely on these defaults

Reply:
you can use:

```
export PORTAL_SOURCE_DIR ="${PORTAL_SOURCE_DIR:-${PWD}}"
```

Reply:
oooh.  so nice.